filters/imagefilter.go

```diff
@@ -53,6 +53,15 @@ func HandleImageResponse(ctx filters.FilterContext, f ImageFilter) {
 }
 
 func handleImageTransform(out *io.PipeWriter, in io.ReadCloser, f ImageFilter) error {
+	var err error
+
```

Comment:
@danpersa I've seen in you originally had both at the same [place](https://github.com/zalando-incubator/skrop/commit/29b63039c7ae3399e4ee1ef3d7a51bd7e36fd730#diff-a174288fd1b708f135648d1d53b03b0aR33) why did you split both?